kp_gain: 1.0
ki_gain: 0.03

pass_through: 0

time_horizon: 3.0
dist_thresh: 0.1
pca_on: 1
